modules/namespace-conditions-for-platform-categories.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * operating/respond-to-violations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="namespace-conditions-for-platform-categories_{context}"]
+= Namespace conditions for platform categories
```

Comment:
Shouldn't it be platform components and not categories? Or am I misunderstanding it? Same for other mentions of categories on this page.

Reply:
I will change it to "components"